.changeset/afraid-rivers-talk.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin-color-picker": patch
+---
+
+Add a gap to `FooterClearButton` slot to prevent the typography from sticking to the color preview
```

Comment:
The name `FooterClearButton` and the concept of slots is not really relevant to people using Comet in their projects

```suggestion
Add missing spacing to the clear-color button of `ColorPicker`
```

Reply:
Changed here: aab5772f03d13cba5232bf2e99c2a6426ef17e38